### PR TITLE
Add livesite settings

### PR DESCRIPTION
**Pull request by ianmah**
Immediate needs, we will probably add more soon?

### CODE REVIEW

File: nw-firestore-schema.ts [hunk 1]
```diff
@@ -105,3 +105,8 @@ interface Root {
   Hackathons: Hackathon[];
   FAQ: FAQ[];
 }
+
+interface Livesite {
+  imgURL: string; // url to logo image
```
Comment (jackyzha0) [inline nw-firestore-schema.ts:110]: should this be hackathonImgUrl instead to make it more clear

Comment (ianmah) [reply nw-firestore-schema.ts:110]: Maybe, technically its possible for it to be something other than a hackathon image, ie a designer could make it say "LIVESITE"?

Comment (jackyzha0) [reply nw-firestore-schema.ts:110]: ah ok, that's fine then

**Review by jackyzha0 — APPROVED** [review]
🐩 nice

**Pull request merged**